contrib/docker/nightly-tester/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:22.04
```

Comment:
I am using 22.04 so that clang is a bit newer. I am happy to use 24.04 too.

Reply:
Whichever you think is most useful